arviz/plots/bpvplot.py

```diff
@@ -44,50 +44,50 @@ def plot_bpv(
     data : InferenceData
         :class:`arviz.InferenceData` object containing the observed and
         posterior/prior predictive data.
-    kind : str, default "u_value"
-        Type of plot to display ("p_value", "u_value", "t_stat").
-        For "p_value" we compute p := p(y* ≤ y | y). This is the probability of the data y being
-        larger or equal than the predicted data y*. The ideal value is 0.5 (half the predictions
-        below and half above the data).
-        For "u_value" we compute pi := p(yi* ≤ yi | y). i.e. like a p_value but per observation yi.
-        This is also known as marginal p_value. The ideal distribution is uniform. This is similar
-        to the LOO-pit calculation/plot, the difference is than in LOO-pit plot we compute
-        pi = p(yi* r ≤ yi | y-i ), where y-i, is all other data except yi.
-        For "t_stat" we compute := p(T(y)* ≤ T(y) | y) where T is any test statistic. See t_stat
-        argument below for details of available options.
-    t_stat : str, float, or callable, default "median"
-        Test statistics to compute from the observations and predictive distributions.
-        Allowed strings are "mean", "median" or "std".
-        Alternative a quantile can be passed as a float (or str) in the
-        interval (0, 1). Finally a user defined function is also
+    kind : {"p_value", "u_value", "t_stat"}, default "u_value"
```

Comment:
```suggestion
    kind : {"u_value", "p_value", "t_stat"}, default "u_value"
```

Putting the default value first. From [numpydoc](https://numpydoc.readthedocs.io/en/latest/format.html#parameters):

> When a parameter can only assume one of a fixed set of values, those values can be listed in braces, with the default appearing first: